resources/iotjs-hardware-tests/arduino/echoboard.ino

```diff
@@ -0,0 +1,118 @@
+#include <SPI.h>
+
+const int GPIO_IN = 7;
```

Comment:
What is the purpose of this file? Is this the application which do the signal echo on the non-tizen side?

Reply:
This file is the code for the arduino echo board, did you read the documentation? https://github.com/tosyu/js-remote-test/blob/786f319ddcdc458dc1d3236ff6a050e4a7ea1ee7/resources/iotjs-hardware-tests/docs/API-Hardware-Echo-Tests.md